src/modules/transaction/components/UnconfirmedTransactionDataTable.vue

```diff
@@ -274,8 +274,7 @@ export default defineComponent({
       txnQueryParams.updateFieldOrder(blockDescOrderSortingField);
 
       let transactionSearchResult = await dashboardService.searchTxns(transactionGroupType.UNCONFIRMED, txnQueryParams);
-
-      let formattedTxns = await dashboardService.formatUnconfirmedMixedTxns(transactionSearchResult.transactions);
+      let formattedTxns = await dashboardService.formatUnconfirmedMixedTxns(transactionSearchResult.transactions);``
```

Comment:
??

Reply:
I think there is 2 extra ` character. likely added accidently